test/Microsoft.AspNet.Mvc.TestCommon/TestFileChangeToken.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace Microsoft.Extensions.Primitives
 {
-    internal class TestFileChangeToken : IChangeToken
+    public class TestFileChangeToken : IChangeToken
```

Comment:
Internal types + strong naming doesn't work with Moq NetCore now. I don't see us referencing one test project from another so not seeing how this would be problematic. If it is, we can change this into a regular project instead.